platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -113,6 +113,24 @@ Previously, the display name could be obtained only by parsing the operation dis
 Additionally, for JUnit5 and Spock, we updated the test descriptor for dynamic and parameterized tests to include information about the class name and method name containing the test.
 These enhancements enable IDEs to offer improved navigation and reporting capabilities for dynamic and parameterized tests.
 
+#### Filter output in JUnit XML test reports
+
+The new [`includeSystemOutLog` and `includeSystemErrLog` options](userguide/java_testing.html#junit_xml_configuration_output_filtering) allow for controlling whether or not output written to standard output and standard error output during testing is included in JUnit's XML test reports.
+Disabling these options can be useful when running a test task results in a large amount of standard output or standard error data that is not relevant for testing, or to preserve disk space when running jobs on CI.
+
+Set these options by configuring the
+link:{javadocPath}/org/gradle/api/tasks/testing/JUnitXmlReport.html[JUnitXmlReport] options block.
```

Comment:
These types of links only work in asciidoc. We have to use markdown style links.